PhysicsTools/NanoAOD/plugins/Run3ScoutingParticleToRecoPFCandidateProducer.cc

```diff
@@ -0,0 +1,286 @@
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/Scouting/interface/Run3ScoutingParticle.h"
+#include "DataFormats/Common/interface/OrphanHandle.h"
+
+#include "SimGeneral/HepPDTRecord/interface/ParticleDataTable.h"
+#include "fastjet/contrib/SoftKiller.hh"
+
+class Run3ScoutingParticleToRecoPFCandidateProducer : public edm::stream::EDProducer<> {
```

Comment:
we think it would be preferable to have the scouting specific plugins defined in their own directory, maybe PhysicsTools/Scouting would be a useful addition to CMMSW repo. please suggest other possible location

Reply:
`PhysicsTools/Scouting` sounds good to me. Does anybody else disagree?

Reply:
I think it would make sense to have a `PhysicsTools/Scouting`